RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@
 * [**] Add media fitlers to the Site Media screen [#22096]
 * [*] The "aspect ratio" mode on the Site Media screen is now also available on iPhone via the new title menu [#22096]
 * [**] Update the classic editor to use the new Photos and Site Media pickers [#22060]
+* [**] [internal] Remove WPMediaPicker dependency [#22103]
```

Comment:
wdyt of combining this with the item below this one?

Reply:
I changed the entry:

```
* [*] [internal] Rework Tenor (Free GIF) and Stock Photos (Free Photos) pickers [#22066, #22074]
```

The goal was to highlight the fact that Tenor and Stock photos changed. So it's more than just about removing WPMediaPicker, and it needs some testing.